js/pokemon.js

```diff
@@ -510,9 +531,11 @@ function checkPokemonLoaded() {
     if(!loadedImage.complete || loadedImage.naturalWidth == 0 || loadedImage.naturalHeight == 0) {
     
         if(++consecutiveLoadFails < 3) {
-            document.getElementById('nextCountdown').innerHTML = 'This is taking a while to load. Do you want to try loading another one? It won\'t affect your streak. <a href="#" onclick="newPokemon();">Load a new Pok&eacute;mon?</a>';
+            jQuery.data($('#nextCountdown')[0]).lang = "loadfail";
```

Comment:
I think what you're looking for here is:

```
$('#nextCountdown').data('lang', 'loadfail')
```

The jQuery.data function you're using works a bit differently, and won't affect the data-lang attribute